geometries/quaternions/rotation_make.py

```diff
@@ -1,29 +0,0 @@
-import os
```

Comment:
This is one of the routines that wrote the contents of an Aspera CK to a text file; the update to rotation_compare.py above makes this obsolete.